packages/mds-ingest-service/repository/migrations/1627427307591-AddEventsRowIdColumn.ts

```diff
@@ -1,6 +1,8 @@
 import { MigrationInterface, QueryRunner } from 'typeorm'
 
 export class AddEventsRowIdColumn1627427307591 implements MigrationInterface {
+  name = 'AddEventsRowIdColumn1627427307591'
```

Comment:
This is a bit of a stab in the dark - this migration is running fine locally without this change (and jest tests are still passing). However, it's the only difference I noticed between this migration and all the others.

Reply:
Yeah this is definitely the issue. I noticed it was missing in your earlier PR but thought it would work because the build passed. Should have commented. This happened in another PR a few months ago too.

Reply:
webpack strips out the class names, yup!